pkg/endpointmanager/gc_test.go

```diff
@@ -39,6 +39,12 @@ func (s *EndpointManagerSuite) TestmarkAndSweep(c *C) {
 	for _, id := range allEndpointIDs {
 		ep := endpoint.NewEndpointWithState(s, s, testipcache.NewMockIPCache(), &endpoint.FakeEndpointProxy{}, testidentity.NewMockIdentityAllocator(nil), id, endpoint.StateReady)
 		mgr.expose(ep)
+
+		/*=======
```

Comment:
conflict left in tree-

mostly relevant to check `expose` err

Reply:
Fixed.